tests/integration/cmor/_fixes/access/test_access_esm1_5.py

```diff
@@ -211,7 +211,7 @@ def test_only_latitude(monkeypatch, cubes_2d):
 
 def test_only_longitude(monkeypatch, cubes_2d):
     """Test fix."""
-    fix = get_fix_allvar('Amon', 'mon', 'tas')
+    fix = get_fix_allvar('Amon', 'mon', 'pr')
```

Comment:
Why did you change the test to `pr`. Is it not working for `tas`?

Reply:
since we have a Tas class, so `tas` will go through class Tas and AllVars, so we have a test especially for `tas` below, in this part I think it's better to have a general variable which only go through `AllVars` for those coordinates test.